packages/gitgraph-core/src/__tests__/gitgraph.getRenderedData.branchesPaths.test.ts

```diff
@@ -438,4 +439,49 @@ describe("Gitgraph.getRenderedData.branchesPaths", () => {
     expect(result[1][0].name).toBe("feat");
     expect(result[1][0].computedColor).toBe("#000000");
   });
+
+  it("should generate branches paths if I define branches order function", () => {
```

Comment:
👍 for adding a test to illustrate the behavior!

I think we could simplify the test a bit, just to focus on the important part. But no worries, I'll refactor the code a bit myself after merging, there is no big deal ;)